Dockerfile

```diff
@@ -1,11 +1,7 @@
-FROM node:5.8.0
+FROM node:8.1.0
```

Comment:
I've had an old branch where I started trying alpine. Maybe some of this can help you out: https://gist.github.com/c47bf72aa18f61567db8a473dee3dd13